### PR TITLE
refactor: deprecate `$` prefix from `ws.properties` keys

### PR DESCRIPTION
**Pull request by SuperchupuDev**
**Please describe the changes this PR makes and why it should be merged:**
- https://github.com/discordjs/discord.js/pull/8094#issuecomment-1155549091
- https://github.com/discord/discord-api-docs/pull/5067

**Status and versioning classification:**
- I know how to update typings and have done so, or typings don't need updating

### CODE REVIEW

**Comment by almeidx** [comment]
As is, this is a breaking change

**Comment by iCrawl** [comment]
We should do this in a non-breaking way, yes. So accept both but only send the "new" way.

**Comment by SuperchupuDev** [comment]
alright then

**Comment by almeidx** [comment]
🤔 Can you deprecate the prefixed keys?

File: typings/index.d.ts [hunk 1]
```diff
@@ -6016,8 +6016,14 @@ export interface WebSocketOptions {
 }
 
 export interface WebSocketProperties {
+  os?: string;
+  browser?: string;
+  device?: string;
+  /** @deprecated Use `os` instead */
```
Comment (Jiralite) [inline typings/index.d.ts:6022]:
```suggestion
  /** @deprecated Use {@link os} instead. */
```

File: typings/index.d.ts [hunk 2]
```diff
@@ -6016,8 +6016,14 @@ export interface WebSocketOptions {
...
   $os?: string;
+  /** @deprecated Use `browser` instead */
```
Comment (Jiralite) [inline typings/index.d.ts:6024]:
```suggestion
  /** @deprecated Use {@link browser} instead. */
```

File: typings/index.d.ts [hunk 3]
```diff
@@ -6016,8 +6016,14 @@ export interface WebSocketOptions {
...
   $browser?: string;
+  /** @deprecated Use `device` instead */
```
Comment (Jiralite) [inline typings/index.d.ts:6026]:
```suggestion
  /** @deprecated Use {@link device} instead. */
```

**Review by Jiralite — COMMENTED** [review]
These `@link`s will point to the correct definition in the typings.

**Review by vladfrangu — APPROVED** [review]

**Review by iCrawl — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

**Comment by JosPoka** [comment]
@SuperchupuDev Could you check the failing check? (vercel)

**Comment by kyranet** [comment]
@JosPoka Vercel only passes on PRs pointing to main, it'll always fail for PRs to v13, don't mind it.

**Review by vladfrangu — APPROVED** [review]

**Review by SpaceEEC — APPROVED** [review]

**Pull request merged**